src/mapeo-project.js

```diff
@@ -554,34 +554,24 @@ export class MapeoProject extends TypedEmitter {
    *  @param {string} opts.configPath
    */
   async importConfig({ configPath }) {
-    // Commenting until #417 and #418 get merged
-    // // check for already present fields, icons and presets and delete them if exist
-    // const alreadyPresets = await this.preset.getMany()
-    // if (alreadyPresets.length !== 0) {
-    //   for (const preset of alreadyPresets) {
-    //     const { fieldIds, iconId, versionId: presetVersionId } = preset
-    //     // delete fields
-    //     for (const fieldId of fieldIds) {
-    //       const field = await this.field.getByDocId(fieldId)
-    //       const { deleted } = await this.field.delete(field.versionId, field)
-    //       if (!deleted) throw new Error('error deleting field from db')
-    //     }
-    //     // delete icon
-    //     const icon = await this.#iconApi.dataType.getByDocId(iconId)
-    //     const { deleted: deletedIcon } = await this.#iconApi.dataType.delete(
-    //       icon.versionId,
-    //       icon
-    //     )
-    //     if (!deletedIcon) throw new Error('error deleting icon from db')
-
-    //     // delete preset
-    //     const { deleted: deletedPreset } = await this.preset.delete(
-    //       presetVersionId,
-    //       preset
-    //     )
-    //     if (!deletedPreset) throw new Error('error deleting preset from db')
-    //   }
-    // }
+    // check for already present fields and presets and delete them if exist
```

Comment:
does it make sense to do this here instead of moving it outside to a function in `src/config-imprort.js`?  I left it here since we are using a couple of methods of the class and it felt weird passing them to the function as params...

Reply:
I'm fine with it here. Could also add a `deleteAll()` method to `DataType` or we could do a quick helper:

```
async function deleteAll(dataType) {
  const deletions = []
  for (const doc of await dataType.getMany()) {
    deletions.push(dataType.delete(doc.versionId))
  }
  return Promise.all(deletions)
}
```